wiki/Tournaments/CMT_4K/2017/en.md

```diff
@@ -53,9 +53,9 @@ Update after group stage starts.
 
 17, June 15:00 (UTC+8)  "The Elder" vs "The Cutting-edge" (4v4）
 
-"The Elder" Team: **gpf19851210** IDSxianyu Cideisbiantai Scslag hans1999 w765333805
+"The Elder" Team: **[gpf19851210](https://osu.ppy.sh/users/711748)** [IDSxianyu](https://osu.ppy.sh/users/7403556) [Cideisbiantai]() [Scslag](https://osu.ppy.sh/users/6705256) [hans1999](https://osu.ppy.sh/users/6679329) [w765333805](https://osu.ppy.sh/users/6423706)
```

Comment:
I prefer the new one since the URL is more meaningful

If needed, I can change all of these URL to the new style